src/app/public/modules/autocomplete/autocomplete.component.html

```diff
@@ -51,6 +49,17 @@
         {{ noResultsFoundText || ('skyux_autocomplete_no_results' | skyLibResources) }}
       </div>
     </div>
+    <div *ngIf="showAddButton"
+      class="sky-autocomplete-actions"
+    >
+      <button *ngIf="showAddButton"
```

Comment:
Just a note - this `ngIf` isn't technically needed at this phase because of the one on the outer `div`. However, it will be needed as a part of the next phase when we add the "Show all" button to the actions container and so I just went ahead and added it here now so that I don't forget it later.